### PR TITLE
New Button and Small Changes

### PR DESCRIPTION
**Pull request by darmuh**
Sending PR from fork

### CODE REVIEW

File: LethalConfig.zip
Comment (AinaVT) [inline LethalConfig.zip:1]: Could you remove this zip file with the compiled plugin?

File: README.md
Comment (AinaVT) [inline README.md:1]: Could you remove this or move this to the CHANGELOG for a new version?

File: Assets/Scenes/TestDarmuh.unity
Comment (AinaVT) [inline Assets/Scenes/TestDarmuh.unity:1]: Could you remove this and the associated meta file?

**Review by AinaVT — CHANGES_REQUESTED** [review]
Nice stuff! Just waiting for you to address the comments, mainly just to remove unnecessary files (like the zip with the binaries).

File: LethalConfig.zip
Comment (darmuh) [reply LethalConfig.zip:1]: Will do, only added them so I could link others to test it out (without posting to thunderstore)

File: README.md
Comment (darmuh) [reply README.md:1]: Changelog should have the same changes mentioned already. Will remove from readme

File: Assets/Scenes/TestDarmuh.unity
Comment (darmuh) [reply Assets/Scenes/TestDarmuh.unity:1]: Removing them from the repository, was using this scene to test my additions/changes (to not mess with other scenes)

**Comment by darmuh** [comment]
Glad to see you found no real issues! I've made the requested changes. Let me know if you need anything else 👍

**Review by AinaVT — APPROVED** [review]
Looks good. Will merge and properly test tomorrow or saturday. If I find any issues i'll get on it.
Thank you!

**Pull request merged**